website/data/docs-navigation.js

```diff
@@ -404,6 +404,7 @@ export default [
       'upgrade-to-1.5.1',
       'upgrade-to-1.5.2',
       'upgrade-to-1.5.3',
+      'upgrade-to-1.6.0',
```

Comment:
Outside the scope of this PR, but I wonder if we should list on descending order and have latest upgrade guide at the top.